aten/src/ATen/native/Loss.cpp

```diff
@@ -148,4 +148,36 @@ Tensor poisson_nll_loss(const Tensor& input, const Tensor& target, const bool lo
 
     return apply_loss_reduction(loss, reduction);
 }
+
+Tensor mse_loss(const Tensor& input, const Tensor& target, int64_t reduction) {
```

Comment:
https://github.com/pytorch/pytorch/blob/8fb756d3b25be6b8f56bbd0d7b52b1f1ed1e7119/torch/nn/functional.py#L2183-L2205

I think we can delete the `broadcast_tensors` call now after this change.